--- openssl11.spec  2023-02-19 22:39:20.000000000 +0000
+++ openssl11.spec  2023-07-01 15:01:32.575258192 +0000
@@ -293,7 +293,7 @@
        zlib enable-camellia enable-seed enable-rfc3779 enable-sctp \
        enable-cms enable-md2 enable-rc5\
        enable-weak-ssl-ciphers \
-       no-mdc2 no-ec2m no-sm2 no-sm4 \
+       no-mdc2 no-ec2m no-sm2 no-sm4 no-tests \
        shared  ${sslarch} $RPM_OPT_FLAGS '-DDEVRANDOM="\"/dev/urandom\""'

 # Do not run this in a production package the FIPS symbols must be patched-in
